jdsol/1.0/_index.md

```diff
@@ -0,0 +1,53 @@
+<!-- Template for the root page of a specification release -->
+---
+title: "Debugging Support for Other Languages Spec 1.0"
+date: 2019-07-18T16:10:38-04:00
+summary: "First Version Spec for DSOL"
+---
+Jakarta Debugging Support for Other Languages provides a mechanism by which 
+programs executed under the JavaTM virtual machine but written in languages 
```

Comment:
The "JavaTM" will not render correctly.  For markdown, I believe it is \&trade;.  Or, you can just use the ™ characters directly.